lib/monitoring/addon/components/cluster-dashboard/component.js

```diff
@@ -91,7 +91,7 @@ export default Component.extend(CatalogUpgrade, {
       scheduler:         false,
       controllerManager: false,
     }
-    const componentStatuses = get(this, 'cluster.componentStatuses');
+    const componentStatuses = get(this, 'cluster.componentStatuses') || [];
```

Comment:
I know nothing about ember computed values, maybe `computed('cluster.componentStatuses.@each.{name,conditions}'` is triggering on null?